netket/vmc_common.py

```diff
@@ -16,3 +16,52 @@ def tree_map(fun, tree):
         return {key: tree_map(fun, value) for key, value in tree.items()}
     else:
         return fun(tree)
+
+import numpy as np
+from netket.utils import jax_available
+if jax_available:
+    from jax.tree_util import tree_flatten, tree_unflatten, tree_map
+    import jax.numpy as jnp
+
+
+def shape_for_sr(grads,jac):
```

Comment:
this and the next function should be indented under if jax_available, right?

Reply:
Yes, I'll fix that now